netbox_metatype_importer/api/nested_serializers.py

```diff
@@ -0,0 +1,18 @@
+from rest_framework import serializers
```

Comment:
Potential issue: The `NestedMetaTypeSerializer` class does not have a `read_only_fields` attribute defined in the Meta class. This could cause unexpected behavior when attempting to deserialize data from an API request. 
Comment: The `NestedMetaTypeSerializer` class should define a `read_only_fields` attribute in the Meta class to ensure that data is deserialized correctly.